client/src/styles/Textbox.module.css

```diff
@@ -0,0 +1,9 @@
+.main {
+    background-color: var(--theme-textbox-background);
+    border: 1px solid var(--theme-border);
+    color: var(--theme-textbox-text);
+    height: var(--theme-md);
```

Comment:
Widgets usually used fixed sizes, instead of being relative to the font size.

Reply:
Done!